plugins/source/awspricing/resources/services.go

```diff
@@ -0,0 +1,320 @@
+package resources
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/cloudquery/cloudquery/plugins/source/awspricing/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+const ROOT_URL = "https://pricing.us-east-1.amazonaws.com"
+
+func Services() *schema.Table {
+	return &schema.Table{
+		Name:                "awspricing_services",
+		Resolver:            fetchServicesTable,
+		PreResourceResolver: getPricingFile,
+		Transform:           transformers.TransformWithStruct(&PricingFile{}, transformers.WithSkipFields("Products", "Terms")),
+		Relations: []*schema.Table{
+			products(),
+			terms(),
+		},
+	}
+}
+
+func fetchServicesTable(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan<- any) error {
+	c := meta.(*client.Client)
+	links, err := getPricingFileLinks(c)
+	if err != nil {
+		return err
+	}
+	res <- links
+	return nil
+}
+
+func validateOffersToSync(c *client.Client, offerCode string) bool {
+	// Can't use glob.Glob because currently it is in plugin-sdk/internal
```

Comment:
The [gobwas/glob](https://pkg.go.dev/github.com/gobwas/glob) package does much more than we need, but I agree having basic glob matching like we have in the SDK would be nice here. IMO it would be fine to copy it as an internal package here as well (not sure we want to expose it as a library from the SDK), but this isn't a blocking comment in any way - starting with only `*` and exact matches is fine for now